### ISSUE TITLE
Feature request: Make use of larger screens in Mura admin with responsive widths...

### ISSUE THREAD

**Issue by JonasEriksson** [blueriver/MuraCMS#3185]
would be useful to allow for wider or adjustable column widths when managing content that requires more than 11-12 characters - see screenshot - managing Custom entities is far more comfortable in any database managing app like Querious than Mura Admin.  
![Image 2020-05-19 at 12 22](https://user-images.githubusercontent.com/701938/82315980-71cf2380-99cc-11ea-9661-dab74deb018a.jpeg)

**Comment by damien-git** [comment]
Agreed. In my plugins I override that CSS to use 100% width. I guess until this is fixed you could use a custom browser stylesheet such as `userContent.css` for Firefox to override that.
```
.content {
    max-width: 100% !important;
}
```
